.eslintrc.json

```
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "trybe-fronted"
```

acho que ta faltando um n aqui
```suggestion
        "trybe-frontend"
```

---

Nossa, muito bem apontado! Fiz mais algumas alterações pra rodar o ESLint, agora vai!